Sprint14

PavelRomashin opened the pull request:
Спринт 14 на проверку

PavelRomashin committed:
Спринт 14, 1 итерация:

PavelRomashin committed:
Спринт 14, 2 итерация, на провеку

Anna-Malinina commented on app/src/main/java/com/example/playlistmaker/PlayerActivity.kt:
```diff
@@ -14,32 +17,54 @@ import java.text.SimpleDateFormat
+        initialization()
         val json = intent.getStringExtra(TRACK)
-        val track = Gson().fromJson(json, Track::class.java)
+        track = Gson().fromJson(json, Track::class.java)
```
В интент можно передавать объекты напрямую, если они реализуют интерфейс parcelable или serializable (на выбор), используя метод putSerializable или putParcelable (доставать соответственно get). Тогда можно обойтись без ручной десериализации.

PavelRomashin committed:
Спринт 14, 3 итерация, на проверку
Примечание: сделать интент через Serialized или Parcelable не вышло, студия ругается на null
